programs/bazaar/src/lib.rs

```diff
@@ -0,0 +1,583 @@
+use anchor_lang::prelude::*;
+use anchor_spl::{
+    associated_token::AssociatedToken,
+    token::{Mint, Token, TokenAccount},
+};
+use solana_program;
+use solana_program::{account_info::AccountInfo, program::invoke, program::invoke_signed};
+
+use spl_math::checked_ceil_div::CheckedCeilDiv;
+
+declare_id!("6Duew5DzYuBMvRPXTXRML2wvp2EvdPKgBofKhUwxHGQi");
+
+#[program]
+pub mod bazaar {
+    use anchor_lang::system_program::Transfer;
+
+    use super::*;
+
+    pub fn initialize_exhibit(
```

Comment:
The naming convention here is really confusing. Is this function meant to be in this contract or did you forget to delete it?